app/controllers/admin/AssetsController.php

```diff
@@ -1208,6 +1236,7 @@ public function getDatatable($status = null)
         $rows[] = array(
             'checkbox'      =>'<div class="text-center"><input type="checkbox" name="edit_asset['.$asset->id.']" class="one_required"></div>',
             'id'        => $asset->id,
+            'image'          => ($asset->image!='') ? '<img src="/uploads/assets/'.$asset->image.'" height=50 width=50></img>' : (($asset->model->image!='') ? '<img src="/uploads/models/'.$asset->model->image.'" height=40 width=50></img>' : ''),
```

Comment:
Can't use an absolute path here, since some people run Snipe-IT in a sub-directory instead of from domain root. Should use something like:

```
 'image'          => ($asset->image!='') ? '<img src="'.Config::get('app.url').'/uploads/assets/'.$asset->image.'" height=50 width=50>' : (($asset->model->image!='') ? '<img src="'.Config::get('app.url').'/uploads/models/'.$asset->model->image.'" height=40 width=50>' : ''),
```

Also TODO, switch to CSS for image sizing instead of HTML.

Also, what's with the closing `img` tag? lol